### DISCUSSION TITLE
arbitrary height value doesn't work

### DISCUSSION THREAD

**Discussion by Chandler-Zhu** [tailwindlabs/tailwindcss#7677] [Help] [answered]
I use arbitrary value like `h-[30%]` in my project but it doesn't work as I expect
my code: https://play.tailwindcss.com/xVdXtJXF5u
expected result:
![tailwind-gh](https://user-images.githubusercontent.com/61914365/155840324-6df755cb-b00e-4fb1-9992-ec572646f63e.png)

**Comment by adamwathan** [comment, marked as answer]
Hey! The CSS is being generated correctly, it's just your actual approach to the problem won't work. You can't really set heights the way you are trying to set them with CSS grid, better to use flexbox for that:

https://play.tailwindcss.com/pjPv7xPvqg

Alternatively you can use grid but set the row heights to match the split you want:

https://play.tailwindcss.com/v9s2GjBMYH
